<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget {
	background-color: rgb(40,44,52);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
  color:#666666;
  background-color: rgb(40,44,52);
  padding: 1px 15px 1px 3px;
  border:1px solid rgba(228,228,228,1);
  border-radius:5px 5px 0px 0px;
  font: 25 16pt &quot;Microsoft YaHei&quot;;
} 
/*设置下拉框边框*/
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 40px;
    border:none;
}
/*设置下拉框按钮图标*/
QComboBox::down-arrow {
	image: url(:/icons/downarraw.png);
}
QComboBox::down-arrow:hover {
	image: url(:/icons/downarraw_hover.png);
}
QComboBox QAbstractItemView{
    background:rgba(255,255,255,1);
    border:1px solid rgba(228,228,228,1);
    border-radius:0px 0px 5px 5px;
    font: 25 16pt &quot;Microsoft YaHei&quot;;
    outline: 0px;  
}

QComboBox QAbstractItemView::item{
    height:36px;
    color:#666666;
    padding-left:9px;
    background-color:#FFFFFF;
}
QComboBox QAbstractItemView::item:hover{ 
    background-color:#409CE1;
    color:#ffffff;
}
QComboBox QAbstractItemView::item:selected{
    background-color:#409CE1;
    color:#ffffff;
}
QComboBox:on { 
    padding-top: 3px;
    padding-left: 4px;
}
QComboBox::down-arrow:on { 
    top: 1px;
    left: 1px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    margin:4px;
	border-image: url(:/icons/close.png);
}
QPushButton:hover{
	border-image: url(:/icons/close_hover.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QStackedWidget {
    border: 1px solid #cccccc;
    border-radius: 4px;
}
</string>
     </property>
     <widget class="QWidget" name="page_btn">
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: rgba(190, 190, 190, 1);
    background-color: rgba(58, 67, 80, 1);
	border-radius:4px;
    font: normal 15pt Microsoft Yahei;
	border: none;
}
QPushButton:hover{
    color: rgba(190, 190, 190, 1);
    background-color:rgb(75, 75, 75);
	border-radius:4px;
    font: normal 15pt Microsoft Yahei;
	border: none;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}
</string>
       </property>
       <property name="text">
        <string>按钮1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_agree">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>110</width>
         <height>80</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    width: 110px;
    height: 80px;
    color: rgb(0, 255, 255);
	font: 25 14pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/icons/ctrl_cmd.png);
}

QPushButton:hover,QPushButton:pressed, QPushButton:checked {
    width: 120px;
    height: 90px;
    color: rgb(0, 255, 255);
    font: 25 20pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/icons/ctrl_cmd.png);
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_table">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
